src/convert/dataframe.jl

```diff
@@ -1,9 +1,15 @@
 # conversion methods for DataFrames
 
-function rcopy(::Type{T}, s::Ptr{VecSxp}; sanitize::Bool=true) where T<:AbstractDataFrame
+function rcopy(::Type{T}, s::Ptr{VecSxp};
+               normalizenames::Bool=true, 
+               sanitize=nothing) where T<:AbstractDataFrame
+    if sanitize != nothing
```

Comment:
Best to use `!==` (the union splitting optimisation doesn't work quite as well with `!=`)

Reply:
Changed and pushed.